arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTrace.storyboard

```diff
@@ -0,0 +1,277 @@
+                                                    <color key="textColor" systemColor="secondaryLabelColor" red="0.23529411759999999" green="0.23529411759999999" blue="0.26274509800000001" alpha="0.59999999999999998" colorSpace="custom" customColorSpace="sRGB"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="Vfb-sE-cuk" firstAttribute="leading" secondItem="4oS-Mf-8HG" secondAttribute="leadingMargin" id="Cv2-k6-DZo"/>
+                                                <constraint firstItem="Vfb-sE-cuk" firstAttribute="top" secondItem="4oS-Mf-8HG" secondAttribute="topMargin" constant="8" id="Fi6-Tp-k2X"/>
+                                                <constraint firstAttribute="trailingMargin" secondItem="Vfb-sE-cuk" secondAttribute="trailing" id="PLk-ab-eIk"/>
+                                                <constraint firstAttribute="bottomMargin" secondItem="Vfb-sE-cuk" secondAttribute="bottom" constant="8" id="WBl-ea-6x2"/>
+                                            </constraints>
+                                        </tableViewCellContentView>
+                                        <connections>
+                                            <outlet property="conditionsLabel" destination="Vfb-sE-cuk" id="KCX-T9-zTg"/>
+                                        </connections>
+                                    </tableViewCell>
+                                </prototypes>
+                                <connections>
+                                    <outlet property="dataSource" destination="eeV-QV-qKJ" id="oLr-zA-oMG"/>
+                                    <outlet property="delegate" destination="eeV-QV-qKJ" id="kdh-pO-Qzk"/>
+                                </connections>
+                            </tableView>
+                            <toolbar opaque="NO" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="cNs-C6-zew">
+                                <rect key="frame" x="0.0" y="813" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem title="Reset" id="Eec-Mb-rgh">
+                                        <connections>
+                                            <action selector="resetBarButtonItemTapped:" destination="eeV-QV-qKJ" id="kqe-1v-txa"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="RQv-69-HIS"/>
+                                    <barButtonItem systemItem="add" id="ISN-Q1-lte">
```

Comment:
Methinks this should be a system Done button.
![image](https://user-images.githubusercontent.com/2257493/91753491-d2016400-eb7c-11ea-9947-4ead15228ce7.png)